test/functional/services/lib/web_element_wrapper/web_element_wrapper.ts

```diff
@@ -249,11 +252,14 @@ export class WebElementWrapper {
    * @default { charByChar: false }
    */
   async clearValueWithKeyboard(options: TypeOptions = { charByChar: false }) {
-    if (options.charByChar === true) {
+    if (options.charByChar === true || this.browserType === Browsers.InternetExplorer) {
       const value = await this.getAttribute('value');
+      // For IE testing, the text field gets clicked in the middle so
+      // first go HOME and then DELETE all chars
+      await this.pressKeys(this.Keys.HOME);
       for (let i = 0; i <= value.length; i++) {
-        await this.pressKeys(this.Keys.BACK_SPACE);
-        await delay(100);
+        await this.pressKeys(this.Keys.DELETE);
+        // await delay(10);
```

Comment:
@LeeDr I suggest to isolate IE-specific logic and keep existing one (Chrome, FF) as is. In this case we we know that if something breaks it is not because of IE changes.
Smth like:
```
async clearValueWithKeyboard(options: TypeOptions = { charByChar: false }) {
  if (this.browserType === Browsers.InternetExplorer) {
    const value = await this.getAttribute('value');
    // For IE testing, the text field gets clicked in the middle so
    // first go HOME and then DELETE all chars
    await this.pressKeys(this.Keys.HOME);
    for (let i = 0; i <= value.length; i++) {
      await this.pressKeys(this.Keys.DELETE);
    }
  }
.... existing logic
```

Reply:
You're right!  My change broke some Firefox tests.  On IE it seemed best to use HOME and DELETE for each char.  

But for Firefox (I think we just assume the cursor is at the end of the text in the input field) we just use BACK_SPACE for each char.

The BACK_SPACE technique by itself doesn't work in IE because we clicking the mouse in the middle of the field, which might not be the end of the existing value.

Commit coming soon...